### DISCUSSION TITLE
Modernize the editing experience

### DISCUSSION THREAD

**Discussion by sethdaily** [tinacms/tinacms#4480] [Ideas]
A modern web tool should have AI and be as smooth to use as possible.
Missing:
- Search engine optimization AI tool
- Spell check
- Dropdown menu search

Solution: Add the features

**Comment by SGiumelli** [comment]
That's a great idea ! You can build it ?

**Comment by drwharris** [comment]
You only get 1 first impression. Anything to improve the UX is important.

**Comment by Calinator444** [comment]
I'd say the spell checker should be lowest on the priority list. A lot of browsers have built in spell checking (e.g. Edge) and even for browsers that don't users can use Grammarly. I think it's nice to have but there's a lot of work to be done for Tina and it's important to prioritize it all.

**Comment by SamBWagner** [comment]
Agree with this, our two highest priorities should be: Making it easy to implement, making it easy to use. Features like these would work with the latter.
